test_scripts/API/VehicleData/GenericNetworkSignalData/commonGenericNetSignalData.lua

```diff
@@ -133,6 +133,9 @@ local function VehicleDataItemsWithDataTableCreation()
     common.VehicleDataItemsWithData.instantFuelConsumption.APItype = "VEHICLEDATA_FUELCONSUMPTION"
     common.VehicleDataItemsWithData.fuelRange.value = {
       { type = "GASOLINE" , range = 20 }, { type = "BATTERY", range = 100 }}
+    common.VehicleDataItemsWithData.stabilityControlsStatus.value = {
```

Comment:
Should  the stabilityControlsStatus parameters really be added to commonGenericNetSignalData ?

Reply:
@VjKlepikov Yes, this change is required to prevent script fails after current PR will be merged.
Small changes for improve readability was made (see 833b0b2)